subscribers:
  camera_reading:
    topic: /usb_cam/image_raw
    queue_size: 1
  obstacle_reading:
    topic: /tracked_obstacles
    queue_size: 100
  lrc_to_xavier:
    topic: /lrc2xav_msg
    queue_size: 1

publishers:
  lane_coef:
    topic: /lane_msg
    queue_size: 10
  xavier_to_lrc:
    topic: /xav2lrc_msg
    queue_size: 1

LaneDetector:  
  steer_angle: 0.0
  eL_height: 0.2
  e1_height: 1.2563
  trust_height: 0.6667
  lp: 609.3 

ROI:
  dynamic_roi: true
  threshold: 128
  
tcp_ip:
  ip_addr: "tcp://192.168.0.18"
  interface_name: "wlan0"

udp_ip:
  ip_addr: "udp://239.255.255.250"

image_view:
  enable_opencv: false
  wait_key_delay: 1
  enable_console_output: true
